### DISCUSSION TITLE
All connected players the same ip adress in "information"

### DISCUSSION THREAD

**Discussion by seth-space** [LMS-Community/slimserver#1266] [General]
Version:
Lyrion Music Server Versie: 9.1.0 - 1734945993 @ Mon Dec 23 09:31:54 UTC 2024
Hostnaam: lmscommunity-lyrionmusicserver-1
IP-adres van server: **172.17.0.9**

Running in Docker on synology DSM 7.2.2
Network: bridge

Player list:
Boom KEUKEN
Model van muziekspeler: Squeezebox Boom
Type muziekspeler: boom
Firmware: 57
IP-adres van muziekspeler: **172.17.0.1**
MAC-adres van muziekspeler: 00:04:20:1f:68:67
Sterkte van draadloos signaal: 92%
 
RadioTVkamer Squeezebox
Model van muziekspeler: Squeezebox Radio
Type muziekspeler: baby
Firmware: 8.0.1-r16924
IP-adres van muziekspeler: **172.17.0.1**
MAC-adres van muziekspeler: 00:04:20:xxxxx
 
Squeezebox Boom BAD
Model van muziekspeler: Squeezebox Boom
Type muziekspeler: boom
Firmware: 57
IP-adres van muziekspeler: **172.17.0.1**
MAC-adres van muziekspeler: 00:04:20:xxxxx
Sterkte van draadloos signaal: 28%
 
Squeezebox RadioSLK
Model van muziekspeler: Squeezebox Radio
Type muziekspeler: baby
Firmware: 8.5.0-r16962
IP-adres van muziekspeler: **172.17.0.1**
MAC-adres van muziekspeler: 00:04:20:xxxxx
 
Squeezebox Touch
Model van muziekspeler: Squeezebox Touch
Type muziekspeler: fab4
Firmware: 8.5.0-r16957
IP-adres van muziekspeler: **172.17.0.1**
MAC-adres van muziekspeler: 00:04:20:xxxxx
Sterkte van draadloos signaal: 93%

**Comment by michaelherger** [comment]
Most likely that's because you run the container in bridge mode: your LMS is running in a different network than your players. The network addresses are all mapped to the same gateway address. I assume they still work?

**Comment by seth-space** [comment]
Yes they work perfect.
No problem there. Is there an advance in running it not in bridge mode?
@

Op wo 25 dec 2024 om 06:12 schreef Michael Herger ***@***.***
>:

> Most likely that's because you run the container in bridge mode: your LMS
> is running in a different network than your players. The network addresses
> are all mapped to the same gateway address. I assume they still work?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/LMS-Community/slimserver/issues/1262#issuecomment-2561616849>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADAT2MQD66QAH4B6T2MUTS32HI5DBAVCNFSM6AAAAABUEURRNSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNRRGYYTMOBUHE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Ed van Delft

**Comment by michaelherger** [comment]
Some plugins need full network access (host mode), or are easier to use with it (the bridge plugins, Spotty). But other than that bridged should work just fine.

**Comment by seth-space** [comment]
Made it a hostmachine but stll the same ip

Lyrion Music Server Versie: 9.1.0 - 1734945993 @ Mon Dec 23 09:31:54 UTC 2024
Hostnaam: lmscommunity-lyrionmusicserver-HOST
IP-adres van server: 172.17.0.3
HTTP-poortnummer van server: 9000
Besturingssysteem: Debian (Docker) - NL - utf8
Platformarchitectuur: x86_64-linux
Databaseversie: SQLite
Totaal aantal herkende muziekspelers: 6

Boom KEUKEN
Model van muziekspeler: Squeezebox Boom
Type muziekspeler: boom
Firmware: 57
IP-adres van muziekspeler: 172.17.0.1
MAC-adres van muziekspeler: 00:04:20
Sterkte van draadloos signaal: 80%
 
RadioTVkamer Squeezebox
Model van muziekspeler: Squeezebox Radio
Type muziekspeler: baby
Firmware: 8.5.3-r16974
IP-adres van muziekspeler: 172.17.0.1
MAC-adres van muziekspeler: 00:04:20:
 
Squeezebox Boom
Model van muziekspeler: Squeezebox Boom
Type muziekspeler: boom
Firmware: 57
IP-adres van muziekspeler: 172.17.0.1
MAC-adres van muziekspeler: 00:04:20:
Sterkte van draadloos signaal: 93%
 
Squeezebox Boom BAD
Model van muziekspeler: Squeezebox Boom
Type muziekspeler: boom
Firmware: 57
IP-adres van muziekspeler: 172.17.0.1
MAC-adres van muziekspeler: 00:04:20
Sterkte van draadloos signaal: 89%
 
Squeezebox RadioSLK
Model van muziekspeler: Squeezebox Radio
Type muziekspeler: baby
Firmware: 8.5.3-r16974
IP-adres van muziekspeler: 172.17.0.1
MAC-adres van muziekspeler: 00:04:20:
 
Squeezebox Touch
Model van muziekspeler: Squeezebox Touch
Type muziekspeler: fab4
Firmware: 8.5.3-r16974
IP-adres van muziekspeler: 172.17.0.1
MAC-adres van muziekspeler: 00:04:20:
Sterkte van draadloos signaal: 83%
